web-common/src/features/models/utils/model-is-empty/index.ts

```diff
@@ -0,0 +1,11 @@
+import { useRuntimeServiceGetFile } from "@rilldata/web-common/runtime-client";
+
+export function modelIsEmpty(instanceId, modelName) {
```

Comment:
There's a convention that these convenient selectors are named `useXXX()`. In this case, it can be named and called like: `modelFileIsEmpty = useModelFileIsEmpty(...)`. Let's also move this to `features/models/selectors.ts`